test/reshape.jl

```diff
@@ -145,9 +145,10 @@ end
                    variable=["a", "b", "a", "b"], value=[3, 4, 5, 6])
     @test_logs (:warn, "Duplicate entries in unstack at row 3 for key 1 and variable a.") unstack(df, :id, :variable, :value)
     @test_logs (:warn, "Duplicate entries in unstack at row 3 for key (1, 1) and variable a.") unstack(df, :variable, :value)
-    a = unstack(df, :id, :variable, :value)
+    a, b = with_logger(NullLogger()) do
```

Comment:
Why is this needed?

Reply:
I want to suppress outputting a warning to the log (we test above that actually the warning is produced) so that later it is easier to find unexpected warnings if they are printed.

Reply:
Ah, cool! It's always annoying to print warnings during normal operation.